amd64/root/firmware-update.sh

```diff
@@ -32,11 +35,13 @@ echo " "
 echo "Firmware available for flashing:"
 echo " "
 
-ls -1 /usr/share/deCONZ/firmware
+ls -1 "$FW_PATH"
 
```

Comment:
I really want to be able to do a fast update of my raspbee-module. So I would love when I do not need to open my browser and check the latest firmware-file names.
```suggestion
echo "Remote firmware files available for flashing (limit to 10):"
echo " "

echo \$FW_ONLINE|tr \" \" \"\n\"
echo " "
```

Reply:
All the extra `\` aren't ment to be in your solution, are they?